src/assets/sass/protocol/components/_footer.scss

```diff
@@ -18,16 +19,14 @@
 
     a:link,
     a:visited {
-        color: $color-white;
         font-weight: normal;
         text-decoration: underline;
-    }
 
-    a:hover,
-    a:focus,
-    a:active {
-        color: $color-white;
-        text-decoration: none;
+        &:hover,
+        &:focus,
+        &:active {
+            text-decoration: underline;
```

Comment:
Should hover / active link styles have some form of differential treatment here, like `text-decoration: none`? Right now the links in `mzp-c-footer-secondary` don't seem to have any real hover state.

Reply:
Oops, this should be `text-decoration: none;`. Bad copypasta.